### PR TITLE
Fix for `/config` on 8285 based devices

### PR DESCRIPTION
**Pull request by pkendall64**
8285 devices don't have enough memory (RAM) to allocate a 32k JSON object! And we don't need that much on an RX anyway.

### CODE REVIEW

**Review by deadbytefpv — APPROVED** [review]

**Comment by deadbytefpv** [comment]
Tested with both an EP1 and an RP3 that wouldn't show any config parameters on previous master (1d58dca)

**Comment by CapnBry** [comment]
This works on:
* ESP32 Vario
* ESP EP2
* ESP TX (export is 1827 bytes)

However, if everything fits in 2kb on all platforms, why is it allocating 32kb at all? Also the allocation of `options` is also 2kb, so can't that be reduced as well?

**Comment by pkendall64** [comment]
> This works on:
> 
> * ESP32 Vario
> * ESP EP2
> * ESP TX (export is 1827 bytes)
> 
> However, if everything fits in 2KB on all platforms, why is it allocating 32KB at all? Also the allocation of `options` is also 2KB, so can't that be reduced as well?
> 
> I missed where the model configurations export were added, but the button calls them "model configurations" and the import line below them call them "models configuration", so which is it? 😅 It would have been smarter at the time to not spell-out-every-parameter-in-the-longest-way-possible 64 times, to save RAM allocation and minor code space. Can we change this before 3.2? RAM is a limited resource and this is just shooting itself in the foot.

I'm not sure how you'd want to minimise the parameter names. JSON is a bit wordy, but I wanted to keep the file at least semi human readable.

The export code is only running on TX modules, which are ESP32 and they have 520kb of RAM, so plenty to go around.

**Review by JyeSmith — APPROVED** [review]
I did a lot of 3.2 testing with this PR. No probs.

**Comment by CapnBry** [comment]
> The export code is only running on TX modules, which are ESP32 and they have 520kb of RAM, so plenty to go around.

I just feel like there's always "plenty of RAM" until there isn't and someone needs to change the one thing that pushed it over the limit instead of optimizing along the way. Things like the `power` object having "power" included in the sub items is redundant, `telemetry-ratio` could just be "ratio" or even "tlmratio", `dynamic-power` and `boost-channel` being redundant (and we can't even store both of these independently), etc. _Also there is DUPLETX_ESP which is a TX that isn't ESP32_, which I know have been used in a few old controller upgrades / internal modules.

Anyway, that's all only tangential to this so, working is better than not working.

**Review by CapnBry — APPROVED** [review]
